asserts/account_key.go

```diff
@@ -48,12 +48,22 @@ func (ak *AccountKey) Until() time.Time {
 	return ak.until
 }
 
+// Fingerprint returns the fingerprint of the account key.
+func (ak *AccountKey) Fingerprint() string {
+	return ak.pubKey.Fingerprint()
+}
+
 // IsValidAt returns whether the account key is valid at 'when' time.
 func (ak *AccountKey) IsValidAt(when time.Time) bool {
 	return (when.After(ak.since) || when.Equal(ak.since)) && when.Before(ak.until)
 }
 
-func checkPublicKey(ab *AssertionBase, fingerprintName string) (PublicKey, error) {
+// Verify verifies signature is valid for content using the account key.
+func (ak *AccountKey) Verify(content []byte, sig Signature) error {
```

Comment:
How is this change releated to the rest of the branch? Same for the Fingerprint() method above? Is it to provide a similar level of functionality as before (i.e. now that pubkey is no longer public this is needed to get the fingerprint and to do the additional checking?